pilot/pkg/xds/filters/filters.go

```diff
@@ -333,7 +333,7 @@ var (
 								Format: &core.SubstitutionFormatString_TextFormatSource{
 									TextFormatSource: &core.DataSource{
 										Specifier: &core.DataSource_InlineString{
-											InlineString: "%DOWNSTREAM_PEER_URI_SAN%",
+											InlineString: "%UPSTREAM_PEER_URI_SAN%",
```

Comment:
io.istio.peer_principal is the thing we are asserting rbac against, which should be downstream. maybe we need a new key for upstream for stats/logging

Reply:
Ah didn't realize we use it for rbac. I could do _upstream_peer_principal?

Reply:
@kyessenov may have a good opinion here since they did most of this metadata stuff?

Reply:
Not sure I understand the context of this.

Reply:
Did something different